src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write your code here
+const thead = document.querySelector('thead').querySelector('tr');
+const tfoot = document.querySelector('tfoot').querySelector('tr');
+const tbodytr = document.querySelector('tbody').querySelectorAll('tr');
+
+const newHeader = document.createElement('th');
+
+newHeader.textContent = 'Position';
+
+const newFooter = document.createElement('th');
+
+newFooter.textContent = 'Position';
+thead.insertBefore(newHeader, thead.lastElementChild);
+tfoot.insertBefore(newFooter, tfoot.lastElementChild);
+
+tbodytr.forEach((item) => {
+  const bodyPositionCell = item.cells[1].cloneNode(true);
+  const bodyPositionHeader = document.createElement('td');
+
+  bodyPositionHeader.textContent = bodyPositionCell.textContent;
+  item.insertBefore(bodyPositionHeader, item.lastElementChild);
+});
```

Comment:
you can simplify it 
```
const rowElements = table.querySelectorAll('tr');

rowElements.forEach(elem => {
  const copyElem = elem.cells[1].cloneNode(true);

  elem.cells[3].after(copyElem);
});
```

Reply:
kind of hard for me to understand for that moment

Reply:
don't hesitate to ask for some help in the chat